src/web/app/components/question-types/question-edit-answer-form/text-question-edit-answer-form.component.scss

```diff
@@ -1,3 +1,13 @@
 .margin-top-7px {
   margin-top: 7px;
 }
+
+#plain-text-area {
```

Comment:
Please avoid using ID for styling. Why don't use class name?

Reply:
Ok... Thought it is a specific thing down to id, and that I saw uses of ids nearby (in the HTML file, albeit no CSS that decorates them specifically). Will just make into a class then.

Reply:
> (in the HTML file, albeit no CSS that decorates them specifically)

Legacy code from V6 :D

Reply:
Rip... got it. Will just adopt class then:)